packages/office-ui-fabric-react/src/components/Button/DefaultButton/DefaultButton.scss

```diff
@@ -8,51 +8,56 @@
 // --------------------------------------------------
 // Default Button styles
 
-.ms-Button--default {
-  background-color: $ms-color-neutralLighter;
-  border: 1px solid $ms-color-neutralLighter;
-  min-width: $button-core-default-minWidth;
-  color: $ms-color-neutralPrimary;
-
-  &,
-  .ms-Button-label,
-  .ms-Button-icon {
-    height: $button-core-default-height;
-    line-height: $button-core-default-height;
-  }
+.ms-Button{
+  &.ms-Button--default {
```

Comment:
Done, one reason I did it the original way was it makes it more obvious the styles in this case have higher specifivity (more indented).